src/com/ucm/dasi/catan/warehouse/IWarehouse.java

```diff
@@ -0,0 +1,18 @@
+package com.ucm.dasi.catan.warehouse;
+
+import com.ucm.dasi.catan.warehouse.exception.NegativeNumberException;
+import com.ucm.dasi.catan.warehouse.exception.NotEnoughtResourcesException;
+
+public interface IWarehouse {
+
+	/* Methods */
+
+	int getResource(ResourceType resource);
+
+	int getQuantityResource();
+	
+	void substract(Warehouse resourcesToPay) throws NotEnoughtResourcesException ;
+
+	void add(Warehouse resourcesToPay) throws NegativeNumberException;
```

Comment:
Could it be possible to sort the methods? (asc)

Reply:
Yes, of course. For the next features, Can you put some rules or check style or something like to know how to put this type of things?

Reply:
Yes, on my way :)